Tangem/Modules/Shop/ShopCoordinatorView.swift

```diff
@@ -17,14 +17,17 @@ struct ShopCoordinatorView: CoordinatorView {
             SafariView(url: webShopUrl)
         } else {
             NavigationView {
-                if let order = coordinator.shopViewModel?.order { // TODO: refactor. Move initializations from viewModel to coordinator
-                    ShopOrderView(order: order)
-                } else if coordinator.shopViewModel?.pollingForOrder == true {
-                    ShopOrderProgressView()
-                } else {
-                    ShopView(viewModel: coordinator.shopViewModel!)
-                        .navigationLinks(links)
+                ZStack {
+                    if let order = coordinator.shopViewModel?.order { // TODO: refactor. Move initializations from viewModel to coordinator
+                        ShopOrderView(order: order)
+                    } else if coordinator.shopViewModel?.pollingForOrder == true {
+                        ShopOrderProgressView()
+                    } else {
+                        ShopView(viewModel: coordinator.shopViewModel!)
```

Comment:
может на всякий случай 
```suggestion
                    } else if let shopViewModel = coordinator.shopViewModel {
                        ShopView(viewModel: shopViewModel)
```

Reply:
Надо бы, я это код не менял, давай сделаю отдельной задачей

Reply:
поменял тут